src/framework/ui/view/uitheme.cpp

```diff
@@ -524,7 +524,7 @@ void UiTheme::drawPrimitive(QStyle::PrimitiveElement element, const QStyleOption
 
     // GroupBox
     case QStyle::PE_FrameGroupBox: {
-        drawRoundedRect(painter, option->rect, 3, QBrush("#03000000"), QPen(strokeColor(), borderWidth()));
+        drawRoundedRect(painter, option->rect, 3, QBrush("#03000000"), QPen(strokeColor(), 1));
```

Comment:
Is this intentional? Looks like you are replacing a calculated value with a "magic number".

Reply:
Ah yes this is intentional, since borderWidth is 1 for HC themes and 0 for general ones.
The border width around group boxes in the styles dialog needs to be 1 for general themes as well and not just the high contrast ones. 
Perhaps a more appropriate line would be:
```c++
drawRoundedRect(painter, option->rect, 3, QBrush("#03000000"), QPen(strokeColor(), configuration()->isHighContrast() ? borderWidth() : 1));
```
This would also account for further changes to High-Contrast borderWidth should someone choose to modify those values...

Reply:
If the intent is *"follow `borderWidth()` where possible but the element must always have a border"* then I think this would be preferable:

```qml
borderWidth() > 1 ? borderWidth() : 1
```

Or even better:

```qml
Math.max(borderWidth(), 1)
```

I think I saw a few other place where you might want to do something similar.

It's best not to use High Contrast as a way to check for a non-zero border because we might have more themes in the future with different border sizes, or make the border size configurable outside of the High Contrast theme, etc.

Reply:
Fixed!